packages/cookie-theme-toggler/src/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { ThemeName } from '@lidofinance/theme'
+import { COOKIE_THEME_MANUAL_KEY } from './constants'
+
+export const getThemeNameFromCookies = (): ThemeName | null => {
+  const match = document.cookie.match(
+    new RegExp('(^| )' + COOKIE_THEME_MANUAL_KEY + '=([^;]+)')
+  )
+
+  if (!match) {
+    return null
+  }
+
+  return match[2] as ThemeName
+}
```

Comment:
1. should not be some `typeof document` validation here?
2. please double-check, as I can be missing something 

```suggestion
const cookieThemeMatcher = new RegExp(`(^| )${COOKIE_THEME_MANUAL_KEY}=([^;]+)`)
export const getThemeNameFromCookies = (): ThemeName | null =>
  document.cookie.match(cookieThemeMatcher)?.[2] ?? null
```

Reply:
We use this function in 'if (typeof window !== 'undefined') {'
https://github.com/lidofinance/ui/pull/346/files#diff-377f43adac8a0c6a51951621d47a102b023109a07ea5400b35620f9b60e0ecd0R40

but I agree with you that we should add typeof document validation here. I will make that